server/app/services/mongo_pubsub.rb

```diff
@@ -100,24 +100,28 @@ def publish(channel, data)
   # @param [String] channel
   # @param [Hash] data
   def self.publish(channel, data)
-    @supervisor.actors.first.publish(channel, data)
+    @supervisor.actors.first.publish(channel, data) if self.started?
   end
 
   # @param [String] channel
   # @param [Hash] data
   def self.publish_async(channel, data)
-    @supervisor.actors.first.async.publish(channel, data)
+    @supervisor.actors.first.async.publish(channel, data) if self.started?
   end
 
   # @param [String] channel
   # @return [Subscription]
   def self.subscribe(channel, &block)
-    @supervisor.actors.first.subscribe(channel, block)
+    @supervisor.actors.first.subscribe(channel, block) if self.started?
```

Comment:
Now instead of exploding, it just silently does not subscribe, and gets stuck instead?